app/src/main/java/com/github/openwebnet/repository/impl/EnvironmentRepositoryImpl.java

```diff
@@ -102,4 +106,19 @@ public EnvironmentRepositoryImpl() {
         });
     }
 
+    @Override
+    public Observable<Void> delete(Integer id) {
+        return Observable.create(subscriber -> {
+            try {
```

Comment:
We can try, when deleting it will look for
```java
@Required
 private Integer environmentId;
```
and is only in the concrete class, so maybe it works for DomoticModel, but I never tried.